apps/bare-expo/android/app/build.gradle

```diff
@@ -76,9 +76,12 @@ import com.android.build.OutputFile
 
 def projectRoot = rootDir.getAbsoluteFile().getParentFile().getAbsolutePath()
 
+def androidSchemes = gradle.ext.expoGetConfig('android.scheme', [ "json": true ])
+println "android schemes: ${androidSchemes}"
+
 project.ext.react = [
     entryFile: ["node", "-e", "require('expo/scripts/resolveAppEntry')", projectRoot, "android"].execute(null, rootDir).text.trim(),
-    enableHermes: (findProperty('expo.jsEngine') ?: "jsc") == "hermes",  // clean and rebuild if changing
+    enableHermes: (gradle.ext.expoGetConfig("android.jsEngine") ?: gradle.ext.expoGetConfig("jsEngine") ?: "jsc") == "hermes",  // clean and rebuild if changing
     // FIXME: make CI passed first and should rollback to original solution when https://github.com/facebook/react-native/pull/32983 landed.
     // cliPath: new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).getParentFile().getAbsolutePath() + "/cli.js",
     cliPath: projectDir.toPath().relativize(new File(["node", "--print", "require.resolve('react-native/package.json')"].execute(null, rootDir).text.trim()).getParentFile().toPath()).toString() + "/cli.js",
```

Comment:
## Issue 2

This one is less important but the benefit of using the mods was that you could write any value to the auxiliary file, whereas writing directly means we will most likely be limited to values that are upstreamed like `ios.jsEngine`.

As an example, [here](https://github.com/expo/config-plugins/blob/1648e8598b959bf8140a2b984af0b8bb85a7e724/packages/ffmpeg-kit-react-native/src/withCocoaPodsImport.ts#L13) we can freely move from plugin props to `ios/Podfile.properties.json` to `ios/Podfile`.

This could potentially be fixed by using `expo.extras` but that field is used for a bunch of things like exposing values publicly to the runtime (i.e. environment variables).